tests/test_georaster_tiling.py

```diff
@@ -102,6 +107,10 @@ def raster_for_test(cls):
                                         0.0, -1.0000599330133702, 4594461.485763356),
                           crs={'init': 'epsg:3857'})
 
+    @classmethod
+    def raster_for_test_wgs84(cls):
+        return cls.raster_for_test().reproject(dst_crs=WGS84_CRS, resolution=1.0)
```

Comment:
this line make the test runs slowly, do you have a suggestion?

Reply:
in this case we cannot run tests without Internet access